docs/rules/self-closing-comp.md

```diff
@@ -1,4 +1,4 @@
-# Prevent extra closing tags for components without children (self-closing-comp)
+# Prevent extra closing tags for components without children (react/self-closing-comp)
```

Comment:
woops, I wasn't supposed to add the prefix on the title. shall we proceed with this? 😛